### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by howardlesshippin** [saiwal/Utsukta-hub-themes#35]
Alt Text

An easy way to add Alt Text to images has never existed in Hubzilla, despite the fact that it is nowadays necessary to add it across the Fediverse and some fediverse software will not even agree to post images that lack Alt Text. So I was pleased to see that you have added a better option for it in the Solidified Theme. Unfortunately, I couldn't get it to work. 

Reproducing:

I added an image by clicking on the img icon (this was a hot-linked image outside hubzilla).
I click on the photo, and the Alt Text dialogue box pops up.
I add the text.
I click outside the dialogue box (as there is no SAVE button)
I save the post.
Result: there was no Alt Text.

Expected behaviour:
The Alt Text would appear for the image after posting.

Was I doing something wrong?
(In an additional test, I was able to add the Alt Text manually, after clicking on the Source button.)

**Desktop (please complete the following information):**
 - MX Linux
 - La bonne heure hub
 - Browser: Vivaldi 8.14
 -  Hubzilla version 11.4
 - Theme version  ? 2026-08-23

**Comment by saiwal** [comment]
reworked and fixed the bug, should work now

**Comment by saiwal** [comment]
alt text can be added both by clicking the alt button in the attachment bar or directly by clicking on the image.

**Comment by howardlesshippin** [comment]
I'm not _completely_  sure that I am working on the updated version, but it still does not work for me. (https://labonneheure.ch/display/8b300d14-53ac-4801-baa6-c2d09655a057)
I filled in the alt-txt but nothing is being finalized or posted.  When I go in to edit the post, I can still see the Alt - Txt in the dialogue box, but it does not transfer to the post.

**Comment by saiwal** [comment]
your hub hasn't updated yet to the updated version.  i've added theme version in footer to better track releases. you should see `Theme: Solidified v0.1.1-beta` in footer when your hub gets updated to the latest release.
